src/style.css

```
@@ -0,0 +1,121 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap);
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  width: 200px;
+  border: 1px #f3f3f3 solid;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__img_link {
+  margin: 32px 0 40px;
+}
+
+.card__name_link {
+  text-decoration: none;
+  color: #060b35;
+  width: 166px;
+  margin-bottom: 4px;
+}
+
+.card__name_link_text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__product_code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  width: 166px;
+  margin-bottom: 16px;
+  margin-top: 0;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  align-content: flex-end;
+  max-width: 166px;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.card__reviews_link {
+  font-style: normal;
+  text-decoration: none;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  margin-bottom: 16px;
+}
+
+.card__price_text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  margin: 0;
+}
+
+.card__price_value {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+}
+
+.card__buy_button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
+  height: 38px;
```

```suggestion
  height: 40px;
```
add * {box-sizing: border-box;}